test/e2e/networkpolicy_test.go

```diff
@@ -199,10 +199,10 @@ func (data *TestData) setupDifferentNamedPorts(t *testing.T) (checkFn func(), cl
 		// Both clients can connect to both servers.
 		for _, clientName := range []string{client0Name, client1Name} {
 			if err := data.runNetcatCommandFromTestPod(clientName, server0IP, server0Port); err != nil {
-				t.Fatalf("Pod %s should be able to connect %s:%d, but was not able to connect", clientName, server0IP, server0Port)
+				t.Fatalf("Pod %s should be able to connect %s %d, but was not able to connect", clientName, server0IP, server0Port)
```

Comment:
Why do we need to update this? I think IP:Port is a convention format?

Reply:
Since it is a log message, people can understand `<ip> <port>`. Then, maybe there's no need to turn it into `<ip>:<port>` or `[<ip>]:<port>`?

Reply:
Thank you @weiqiangt and @antoninbas. I thought it could be like `nc <ip> <port>` but consistency should be more important.
Updated.